site/src/pages/docs/advanced-zipkin.mdx

```diff
@@ -47,7 +47,7 @@ Now, you can specify <type://BraveService> using [Decorating a service](/docs/se
 ```java
     import com.linecorp.armeria.common.HttpResponse;
     import com.linecorp.armeria.server.Server;
-    import com.linecorp.armeria.server.brave.BraveService;
+    import com.linecorp.armeria.server.observation.BraveService;
```

Comment:
Can you revert this since the class doesn't exist? 😅 
I'll be adding a separate page on tracing later